booking-up-for-beauty/damoreno/booking_up_for_beauty.go

```diff
@@ -0,0 +1,46 @@
+package booking
+
+import (
+    "fmt"
+    "time"
+)
+
+func parseDate(layout, date string) time.Time {
+    parsedDate, err := time.Parse(layout, date)
+
+    if err != nil {
+	panic("The suplied date is not parseable")
```

Comment:
Not sure if the real error (the one `err` is holding) is printed that way, but if it's not you can take the value from err as a string by `err.Error()` and add it to your custom message.